packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -945,7 +944,7 @@ export class LocalDetachedFluidDataStoreContext
             scope,
             createSummarizerNode,
             bindChannel,
-            snapshotTree,
+            undefined,
```

Comment:
Is this related, or incorrect merge conflict? I'm not sure I see relationship to your changes.

Reply:
`LocalDetachedFluidDataStoreContext`'s snapshotTree is always `undefined` (ending up being non-'existent') so I just thought to make this more explicit..